modules/RockRuby.cmake

```diff
@@ -266,6 +266,11 @@ ELSEIF(NOT RUBY_EXTENSIONS_AVAILABLE)
        OUTPUT_VARIABLE RUBY_CFLAGS)
     STRING(REPLACE "\n" "" RUBY_CFLAGS ${RUBY_CFLAGS})
 
+    if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
+        #Clang can't handle this option (04.2014) remove this flag therefore
+        STRING(REPLACE "-fstack-protector-strong" "" RUBY_CFLAGS ${RUBY_CFLAGS})
```

Comment:
And just disabling the flag is kindof simple. Maybe [testing](http://stackoverflow.com/a/3135240/4658481) for it is a better idea?